test.js

```diff
@@ -52,6 +46,16 @@ it('should glob - sync', function () {
 	assert.deepEqual(globby.sync(['!*.tmp', 'a.tmp']), ['a.tmp']);
 });
 
+it('should return [] for all negative patterns - sync', function () {
+	assert.deepEqual(globby.sync(['!a.tmp', '!b.tmp']), []);
+})
+
+it('should return [] for all negative patterns - async', function () {
+	return globby(['!a.tmp', '!b.tmp']).then(function (paths) {
+		assert.deepEqual(paths, []);
+	});
+})
```

Comment:
Let me know if you think these test cases are worthwhile. Wanted to hit the code path to cover the callback reference I missed.